pkgs/applications/office/qownnotes/default.nix

```diff
@@ -19,14 +19,14 @@
 let
   pname = "qownnotes";
   appname = "QOwnNotes";
-  version = "23.6.6";
+  version = "23.7.1";
 in
 stdenv.mkDerivation {
   inherit pname appname version;
 
   src = fetchurl {
-    url = "https://download.tuxfamily.org/${pname}/src/${pname}-${version}.tar.xz";
-    hash = "sha256-UXEnoe8TovzghKtEXDis5Gk93oDEyJhhNXMnGaK5VXI=";
+    url = "https://github.com/pbek/QOwnNotes/releases/download/v${version}/qownnotes-${version}.tar.xz";
+    hash = "sha256-xrt9xeBANcrTf84YR8jLDZkoTAQ4SXrlGsVJPXCefqw=";
   };
```

Comment:
actually since you are now using gitHub for releases you should also change this to the fetchFromGItHub wrapper

Reply:
Many upstreams will not actively maintain GitHub Releases, we must use fetchFromGitHub to get the source code corresponding to a tag/rev
If the upstream officially maintains GitHub Releases, there is no need to use fetchFromGitHub instead. The code provided by Releases may delete unnecessary files such as github ci, which is cleaner
https://github.com/pbek/QOwnNotes/issues/2799

Reply:
Yes, the package I am downloading is specially prepared. Things are copied around, removed, binary translations generated, things set in header files. This is the official source package all other packaging system, like snap, flatpack, Ubuntu Launchpad ppa, aur, gentoo rebuild,... Are using.